src/main.rs

```diff
@@ -57,11 +63,58 @@ pub struct Args {
     log_file: Option<String>,
 }
 
+/// Subcommands for 'qmassa'
+#[derive(Subcommand, Debug)]
```

Comment:
I think this will be almost the same with the subcommand structure I added (slightly different with the cmd args being a separate struct).

Reply:
cool I will change that